### ISSUE TITLE
yvanzo’s All shook up links

### ISSUE THREAD

**Issue by jesus2099** [jesus2099/konami-command#488]
I am testing work on this @yvanzo big set of features #244.

**Comment by jesus2099** [comment]
It’s superb that #244 × 2 = #488.

**Comment by jesus2099** [comment]
Removed the special _merge_ technique from OP:

> ```sh
> git checkout master
> git checkout #244_y-van-z_all-shook-up-links mb_INLINE-STUFF.user.js
> ```
>
> (it seems that [using ``--patch``](https://towardsdatascience.com/merging-only-one-file-from-a-git-branch-patching-3a9b5a9c3fa6) would be if I wanted to pick changes instead of taking all file)

I tested this technique for [updating `.gitignore`](https://github.com/jesus2099/konami-command/commit/a415ccab15b7b89c4ebbf54fbaf6de77e7e01a9f) and it does not show as a branch merge.
I want to show the Yvanzo branch merging into master, explicitely, in history.

**Comment by jesus2099** [comment]
Hi @yvanzo,

Could you tell me if I understood right in my recent fbc901543b44384c58fcf753175eff2777a3789a?

Every time I update this #244 branch, GitHub suggests to create a pull request (PR, merge request).
Do you think it would be better, maybe?

**Comment by yvanzo** [comment]
Hi @jesus2099,

Adding `parentNode` seems to be correct but I’m not so sure about the added `sidebar` in this commit. IIRC the idea was to insert them next to other search links, maybe even in the `searchLinks` list? I don’t see them appearing anyway. Wasn’t there a dialog to allow setting them too?

About the pull request workflow, it’s only useful when you want feedback from someone else before merging.

Thanks again to keep caring about user scripts!

**Comment by jesus2099** [comment]
Yes, there is little bit of inconsistencies between the (I guess) similar `addHiddenLinks()`, `addSearchLinksSection(sectionPath, parentNode)` and `addUserLinks(parentNode)`.

I will check that in the future.
User links are empty per default, there used to be an ugly `prompt` to edit them.
They would be good before the standard search links, because they are user specifics, user needs.

**Comment by jesus2099** [comment]
I remove #591 from here, match/include/exclude are not related to this branch merge 9d41b761f32748c0d9425da3e45692ec8ffc6a2c.

**Comment by jesus2099** [comment]
No strong need for ~Loop in all catalogue~, at the moment.
